frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js

```diff
@@ -14,6 +14,7 @@ import FormLabel from 'Components/Form/FormLabel';
 import FormInputGroup from 'Components/Form/FormInputGroup';
 import ProviderFieldFormGroup from 'Components/Form/ProviderFieldFormGroup';
 import styles from './EditNetImportModalContent.css';
+import availabilityOptions from 'Components/Form/AvailabilitySelectInput';
```

Comment:
You don't need this when specifically using the input type for availability select